aws-app-sync/src/test/java/com/baeldung/awsappsync/AwsAppSyncApplicationTests.java

```diff
@@ -0,0 +1,72 @@
+package com.baeldung.awsappsync;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.web.reactive.function.client.WebClient;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@SpringBootTest
+class AwsAppSyncApplicationTests {
+
+    @Test
+    void givenGraphQuery_whenListEvents_thenReturnAllEvents() {
+
+        Map<String, Object> requestBody = new HashMap<>();
+        requestBody.put("query", "query ListEvents {" +
+                "  listEvents {" +
+                "    items {" +
+                "      id" +
+                "      name" +
+                "      where" +
+                "      when" +
+                "      description" +
+                "    }" +
+                "  }" +
+                "}");
```

Comment:
When breaking up a line at an operator boundary, the guidelines say to move the operator to the beginning of the continuation line, rather than leaving it the end of the line before the break.